src/mark.ts

```diff
@@ -47,6 +47,11 @@ export interface MarkDef {
    * A metadata string indicating the role of the mark.
    * This allows users to use `config.<role-name>.*` to customize properties of marks with specific roles.
    * In addition, SVG renderers will add this role value (prepended with the prefix role-) as a CSS class name on the enclosing SVG group (`<g>`) element containing the mark instances.
+   *
+   * __Default value:__ For `bar`, `circle`, `point`, `rule`, `square` and `tick` marks, the role is the mark name (e.g., `role: "bar"` for `bar` mark).
+   * For other marks (which use the same marks in the lower-level Vega), role is `undefined` by default.
```

Comment:
"mark type" instead of "mark name"? The former seems more consistent to me, and avoids confusion with the fact that "name" is a valid property within a unit spec.